src/main/java/com/flansmod/common/FlansMod.java

```diff
@@ -132,10 +186,20 @@ public class FlansMod {
     public static float masterLegModifier = 0.5F;
     public static boolean masterDualWieldDisable = false;
     public static boolean gunDevMode = false;
-
+    
     public static float nameTagRenderRange = 64F;
     public static float nameTagSneakRenderRange = 32F;
     public static float maxHealth = 20;
+    
+    public static boolean enableBlockPenetration = false;
```

Comment:
This would be good to put into teams, honestly - so it can be accessed via command ingame.

Also remove the glass commands as well?

Reply:
It would probably be better to not bury it in /teams unless it's explicitly for teams

Reply:
True, we could start a new command for config and move a lot of the stuff into it. Like explosionsCanBreakBlocks, vehiclesBreakBlocks, vehicleLife e.t.c...

But editable ingame is my point, flansmod.cfg requires a server restart or client restart.

Reply:
I would also not wanna put it in teams, but I like the idea of a config change command. But for now, I would leave it as is

Reply:
Alright, we can consider it later as part of a bigger overhaul of flansmod.cfg